src/theme/SearchBar/index.js

```diff
@@ -138,6 +132,7 @@ const Search = props => {
         )}
         onClick={loadAlgolia}
         onMouseOver={loadAlgolia}
+        placeholder={props.placeholder || "Search"}
```

Comment:
There is not an intl system?

Reply:
Did you mean internationalization system?
No, not at the moment.